core/src/main/java/org/elasticsearch/monitor/os/OsStats.java

```diff
@@ -120,7 +124,9 @@ public static OsStats readOsStats(StreamInput in) throws IOException {
     @Override
     public void readFrom(StreamInput in) throws IOException {
         timestamp = in.readVLong();
-        loadAverage = in.readDouble();
+        if (in.readBoolean()) {
```

Comment:
This change breaks backward compatibility between 2.1 & 2.2 (pull request is labelled v2.2.0)